src/olympia/abuse/cinder.py

```diff
@@ -151,8 +151,9 @@ def appeal(self, *args, **kwargs):
 class CinderAddon(CinderEntity):
     type = 'amo_addon'
 
-    def __init__(self, addon):
+    def __init__(self, addon, version=None):
```

Comment:
Note, we don't do anything with this version in general, but we should (very likely) send it - something to consider in https://github.com/mozilla/addons-server/issues/21234